modules/nodes-nodes-working-updating.adoc

```diff
@@ -43,4 +84,157 @@ For example:
 ----
 $ oc label pods --all status=unhealthy
 ----
-
++
```

Comment:
There isn't a plausible declarative equivalent to applying a label to all pods.  Let's remove this.